__tests__/app.test.js

```diff
@@ -127,4 +127,58 @@ describe("GET /api/articles",()=>{
             expect(body.articles).toBeSortedBy("created_at", {descending: true})
         })
     })
+})
+
+describe("GET /api/articles/:article_id/comments",()=>{
+    it("should respond with an array of comment objects with the matching article id",()=>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body})=>{
+            expect(body.comments.length).not.toBe(0)
+            body.comments.forEach((comment)=>{
+                expect(typeof comment.comment_id).toBe("number")
+                expect(typeof comment.votes).toBe("number")
+                expect(typeof comment.created_at).toBe("string")
+                expect(typeof comment.author).toBe("string")
+                expect(typeof comment.body).toBe("string")
+                expect(typeof comment.article_id).toBe("number")
+            })
```

Comment:
I like how you tested this
You could potentially explore .toHaveProperty()
e.g.
.toHaveProperty('votes', expect.any(Number))